gengokit/handler/handler.go

```diff
@@ -23,7 +23,9 @@ import (
 // NewService is an exported func that creates a new service
 // it will not be defined in the service definition but is required
 const ignoredFunc = "NewService"
-const serverTemplPath = "NAME-service/handlers/server/server_handler.gotemplate"
+
+// ServerFile is the path to the service handelr file that handler will render
```

Comment:
"handelr"

It's a mouthful of a sentence anyway, though. How about just "ServerFile is the path to the service handler template file."

It'd probably be better named `ServerHandlerTemplate` or similar. More explicit and more accurate.